tests/test_utils.py

```diff
@@ -77,10 +77,10 @@ def test_load_pipeline_config(self):
         self.pipeline_obj._load_pipeline_config()
         assert self.pipeline_obj.nf_config["dag.enabled"] == "true"
 
-    def test_load_conda_env(self):
-        """Load the pipeline Conda environment.yml file"""
-        self.pipeline_obj._load_conda_environment()
-        assert self.pipeline_obj.conda_config["channels"] == ["conda-forge", "bioconda", "defaults"]
+    # def test_load_conda_env(self):
+    #     """Load the pipeline Conda environment.yml file"""
+    #     self.pipeline_obj._load_conda_environment()
+    #     assert self.pipeline_obj.conda_config["channels"] == ["conda-forge", "bioconda", "defaults"]
```

Comment:
Why uncomment this instead of deleting?

Reply:
It was one for the DSL2 hackathon sprint. I discussed this with @ewels on Slack and we agreed to just comment stuff out for now in favour of deleting it all in one big go. Easier to delete than to figure out where the code has gone given the changes from DSL1 -> DSL2.

Reply:
Added TODO statements https://github.com/nf-core/tools/pull/1109/commits/25a52f8f294714b61cbb07a5874c8b74bb58b341